src/pages/Proposal.tsx

```diff
@@ -262,10 +263,12 @@ const ProposalPage = observer(() => {
     scheme.type === 'WalletScheme'
       ? proposal.submittedTime.plus(scheme.maxSecondsForExecution)
       : bnum(0);
-
-  let proposalCallTexts = new Array(proposal.to.length);
+  
+const proposalCall = async () => {
+  const proposalCallArray = []
   for (var p = 0; p < proposal.to.length; p++) {
-    proposalCallTexts[p] = daoService.decodeWalletSchemeCall(
+    
+    proposalCallArray[p] =   await daoService.decodeWalletSchemeCall(

```

Comment:
2 spaces before await

Reply:
I'll remember to format the document once its finished